spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkAction.java

```diff
@@ -137,14 +140,18 @@ protected Table newStaticTable(TableMetadata metadata, FileIO io) {
     return new BaseTable(ops, metadataFileLocation);
   }
 
-  // builds a DF of delete and data file path and type by reading all manifests
   protected Dataset<FileInfo> contentFileDS(Table table) {
+    return contentFileDS(table, null);
+  }
+
+  protected Dataset<FileInfo> contentFileDS(Table table, Set<Long> snapshotIds) {
     Table serializableTable = SerializableTableWithSize.copyOf(table);
     Broadcast<Table> tableBroadcast = sparkContext.broadcast(serializableTable);
     int numShufflePartitions = spark.sessionState().conf().numShufflePartitions();
 
-    Dataset<ManifestFileBean> allManifests =
-        loadMetadataTable(table, ALL_MANIFESTS)
+    Dataset<Row> manifests = manifestDF(table, snapshotIds);
+    Dataset<ManifestFileBean> manifestsBean =
```

Comment:
nit: Most other vars in this class use `xxxDF` for `Dataset<Row>` or `xxxDS` for `Dataset` of a specific type.

```
Dataset<ManifestFileBean> manifestBeanDS =
    manifestDF(table, snapshotIds)
        .select(...)
```

Reply:
Changed var name and inlined the previous var as suggested.